lib/privacy.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const common = require('./common');
+
+async function privacy (opts) {
```

Comment:
you use async here but the rest of the codebase is still using promises, can you stick to promises here? (You can send a separate PR if you want to switch the entire codebase to async/await)